crates/apollo-mcp-server/src/operations/schema_walker/type.rs

```diff
@@ -0,0 +1,74 @@
+use apollo_compiler::{Schema as GraphQLSchema, ast::Type as GraphQLType};
+use schemars::{Schema as JSONSchema, json_schema};
+use serde_json::{Map, Value};
+
+use crate::custom_scalar_map::CustomScalarMap;
+
+use super::name::Name;
+
+pub(super) struct Type<'a> {
```

Comment:
https://github.com/apollographql/apollo-mcp-server/blob/ea32f7da9871633512af3323068f3286aa216ec6/crates/apollo-mcp-server/src/operations/schema_walker/name.rs#L10-L11

Do you want to add a similar comment here?

```suggestion
/// A GraphQL Type Walker
pub(super) struct Type<'a> {
```

Reply:
Nice catch!